### PR TITLE
Workflow: methods and metrics update

### PR DESCRIPTION
**Pull request by Jieran-S**
Update the workflow files for the current Methods and Metris workflows I used for my project.

### CODE REVIEW

File: workflows/path_configs/methods.yaml [hunk 1]
```diff
@@ -98,119 +98,93 @@ methods:
 # make sure the config_files name is the same as methods name
 config_files:
```
Comment (shdam) [inline workflows/path_configs/methods.yaml:99]: Can't the list of configs be generated automatically?

File: workflows/example_configs/methods_config.yaml [hunk 2]
```diff
@@ -8,26 +8,27 @@ datasets:
...
   - visium_breast_cancer_SEDR
-#  - stereoseq_developing_Drosophila_embryos_larvae
   - visium_chicken_heart
+#  - stereoseq_developing_Drosophila_embryos_larvae
+#  - "stereoseq_mouse_embryo"
 
 seed: 2023
 
 # Methods to run for the pipeline, modify based on your need
 use_methods:
```
Comment (shdam) [inline workflows/example_configs/methods_config.yaml:18]: Could some of the newly merged methods be included?

Comment (Jieran-S) [reply workflows/example_configs/methods_config.yaml:18]: Done!

File: workflows/path_configs/methods.yaml (hunk 1 above)
Comment (shdam) [reply workflows/path_configs/methods.yaml:114]: Sounds sensible 👍

**Review by shdam — APPROVED** [review]

**Pull request merged**